src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1147,6 +1149,12 @@ private int decodeTransfer(
                 if (!sender.fragmented)
                 {
                     assert deliveryId != NO_DELIVERY_ID; // TODO: error
+                    if (transfer.hasAborted() && transfer.aborted() == 1)
+                    {
+                        progress = limit;
+                        server.decoder = decodePlainFrame;
+                        break decode;
+                    }
```

Comment:
This is to handle the case where the first `transfer` fragment is `aborted` right away, so we ignore the message completely, right?